AmplitudeTests/AMPDatabaseHelperTests.m

```diff
@@ -29,6 +29,63 @@ - (void)tearDown {
     self.databaseHelper = nil;
 }
 
+- (void)testGetDatabaseHelper {
+    // test backwards compatibility on default instance
+    AMPDatabaseHelper *dbHelper = [AMPDatabaseHelper getDatabaseHelper];
+    XCTAssertEqual(dbHelper, [AMPDatabaseHelper getDatabaseHelper:nil]);
+    XCTAssertEqual(dbHelper, [AMPDatabaseHelper getDatabaseHelper:@""]);
+    XCTAssertEqual(dbHelper, [AMPDatabaseHelper getDatabaseHelper:[NSNull null]]);
+    XCTAssertEqual(dbHelper, [AMPDatabaseHelper getDatabaseHelper:kAMPDefaultInstance]);
+
+    AMPDatabaseHelper *a = [AMPDatabaseHelper getDatabaseHelper:@"a"];
+    AMPDatabaseHelper *b = [AMPDatabaseHelper getDatabaseHelper:@"b"];
+    XCTAssertNotEqual(dbHelper, a);
+    XCTAssertNotEqual(dbHelper, b);
+    XCTAssertNotEqual(a, b);
+    XCTAssertEqual(a, [AMPDatabaseHelper getDatabaseHelper:@"a"]);
+    XCTAssertEqual(b, [AMPDatabaseHelper getDatabaseHelper:@"b"]);
+
+    // test case insensitive instance name
+    XCTAssertEqual(a, [AMPDatabaseHelper getDatabaseHelper:@"A"]);
+    XCTAssertEqual(b, [AMPDatabaseHelper getDatabaseHelper:@"B"]);
+    XCTAssertEqual(dbHelper, [AMPDatabaseHelper getDatabaseHelper:[kAMPDefaultInstance uppercaseString]]);
+
+    // test each instance maintains separate database files
+    XCTAssertTrue([a.databasePath rangeOfString:@"com.amplitude.database_a"].location != NSNotFound);
+    XCTAssertTrue([b.databasePath rangeOfString:@"com.amplitude.database_b"].location != NSNotFound);
+    XCTAssertTrue([dbHelper.databasePath rangeOfString:@"com.amplitude.database"].location != NSNotFound);
+
```

Comment:
Add a test case for defaultInstance only having com.amplitude.database (without _a)